winter-collector-page.css

```diff
@@ -0,0 +1,207 @@
+html {
+    background-color: rgb(216, 171, 125);
+    color: rgb(71, 39, 7);
+    font-family: Trattatello;
+    font-size: 10px;
+    margin: 30px;
+}
+
+h1, h2 {
+    text-align: center;
+    margin: auto;
+    width: 85%;
+    align: center;
+}
+
+h1 {
+    font-size: 8rem;
+}
+
+h2 {
+    font-size: 4rem;
+}
+
+h3 {
+    color: rgb(77, 71, 63);
+    display: inline;
+    text-align: center;
+}
+
+section.allItemsSection {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin: 20px;
+    font-size: 2rem;
+    justify-content: center;
+    padding: 15px;
+    text-align: center;
+    align-content: center;
+}
+
+ul {
+    list-style-type:none;
+    background: rgb(168, 149, 123);
+    margin: 15px;
+    border-radius: 5px;
+}
+
+div {
+    width: 30%;
+}
```

Comment:
best to give a class to that div, otherwise in future if you add any other div tags they will be affected by this rule and you will have to change it

Reply:
removed the div -- wasn't needed anymore anyway! : ]